test/util.js

```diff
@@ -64,7 +64,6 @@ vows.describe('Tests for config util functions')
         'It can handle recursion': function(util) {
           var result = util.loadFileConfigs(path.join(__dirname, '21-reentrant'));
           assert.ok(result.nested, "did not load nested values");
```

Comment:
I didn't know what to add or test here, so if anything feels missing, let me know